Sources/Development/main.swift

```diff
@@ -10,8 +11,8 @@ server.then { client in
     client.map(parser.parse).map { request in
         return HTTP.Response(status: 200)
     }.then(client.send)
-
-    client.listen()
+    
```

Comment:
can you enabled trim white space only lines in Xcode > Preferences > Text Editing > While Editing